rosidl_generator_py/resource/_msg.py.template

```diff
@@ -72,24 +73,27 @@ class Metaclass(type):
 
 
 class @(spec.base_type.type)(metaclass=Metaclass):
+@[if not spec.constants]@
+    """Message class '@(spec.base_type.type)'."""
+@[else]@
     """
     Message class '@(spec.base_type.type)'.
-@[if spec.constants]@
 
     Constants:
 @[  for constant in spec.constants]@
       @(constant.name)
 @[  end for]@
-@[end if]@
     """
+@[end if]@
 
     __slots__ = [
 @[for field in spec.fields]@
         '_@(field.name)',
 @[end for]@
     ]
-
+@
 @[if len(spec.fields) > 0]@
+
```

Comment:
Correct, the template needs to generate the right amount of empty line before all methods for every possible input. Therefore it now doesn't generically add a newline in 91 but only adds it in 96 before `def__init__`. Later on each field related method also has a leading empty line.